mysql/toddl.go

```diff
@@ -214,6 +215,17 @@ func cvtForeignKeys(conv *internal.Conv, srcTable string, srcKeys []schema.Forei
 				conv.Unexpected(fmt.Sprintf("Can't map foreign key for table: %s, referenced table: %s, column: %s", srcTable, key.ReferTable, col))
 				continue
 			}
+			if _, found := conv.SpSchema[spReferTable].ColDefs[spReferCol]; !found {
```

Comment:
Couldn't this happen for tables as well i.e. the refer table could be different case from the table name?
In other words, should we do the same case translation for tables as well if we can't find the table?

Reply:
I just want to add that we will need to take care of this for other references as well e.g., 1. columns referenced as part of primary keys, 2. table/column names referenced as part of secondary index definitions.

Reference: "When referring to other schema objects in a DDL statement (for example, a column name for a primary key, or table and column names in an index), make sure to use the original case for the name of each entity". 

https://cloud.google.com/spanner/docs/data-definition-language#naming_conventions

Fixing primary key references could be it's own separate PR whereas, fixing secondary index references should be part of the ongoing PR https://github.com/cloudspannerecosystem/harbourbridge/pull/95.

Reply:
added a TODO to handle primary key and index references